tox.ini

```diff
@@ -14,8 +14,8 @@ envlist =
 [testenv]
 commands =
     pytest
-extras =
-    test
+; extras =
+;     test
```

Comment:
why this change?

Reply:
I was trying something .... I had a suspicion it was causing some error. It is not and hence uncommented again.